src/titiler/core/titiler/core/dependencies.py

```diff
@@ -445,3 +446,18 @@ def __post_init__(self):
 
         if self.range:
             self.range = list(map(float, self.range.split(",")))  # type: ignore
+
+
+def TileParams(
+    z: int = Path(..., ge=0, le=30, description="Tiles's zoom level"),
+    x: int = Path(..., description="Tiles's column"),
+    y: int = Path(..., description="Tiles's row"),
+    scheme: Optional[Literal[("xyz", "tms")]] = Query(
+        None, description="Tile Scheme (default to XYZ)."
+    ),
+) -> Tile:
+    """Tile parameters."""
+    if scheme == "tms":
+        y = 2**z - y - 1
```

Comment:
👀 from https://gist.github.com/tmcw/4954720

Reply:
To my understanding it goes that way, since `y_TMS = 2^z - y_XYZ -1` but also `y_XYZ = 2^z - y_TMS -1`. From the Titiler docs, isn't the default TMS? 
![image](https://user-images.githubusercontent.com/15091668/217678975-d509712c-2d37-4521-811c-90fc2c07790e.png)

Reply:
> From the Titiler docs, isn't the default scheme TMS?

there is a difference between `OGC TMS -> TileMatrixSet` and `Osgeo TMS -> Tile Map Service` which is what the scheme is. 

in TiTiler we use the acronyme TMS as the OGC one `TileMatrixSet`

Reply:
Okay, alright, now I get it 👌